test/box-luatest/gh_6199_schema_priv_resolve_existence_check_test.lua

```diff
@@ -0,0 +1,39 @@
+local cluster = require('test.luatest_helpers.cluster')
+local t = require('luatest')
+
+local g = t.group('gh-6199-schema-priv-resolve-existence-check')
+
+g.before_all(function()
+    local helpers = require('test.luatest_helpers')
+
+    g.cluster = cluster:new({})
+
+    local default_box_cfg = {
+        listen = helpers.instance_uri('default'),
+    }
+    g.default = g.cluster:build_and_add_server({alias   = 'default',
+                                                box_cfg = default_box_cfg})
+    g.cluster:start()
+    g.default:exec(function()
+        box.session.su('admin')
+    end)
+end)
+
+g.after_all(function()
+    g.cluster:drop()
+end)
+
+g.test_priv_resolve_existence_check = function()
+    g.default:exec(function()
+        local t = require('luatest')
+
+        local msg = 'role cannot be granted together with a privilege'
```

Comment:
I'd add more test cases, at least one for each privilege. Otherwise LGTM

Reply:
Fixed: added test cases for each privilege, added a test case for all privileges combined.